include/mlir/IR/OpImplementation.h

```diff
@@ -486,6 +501,14 @@ class OpAsmParser {
     result.append(types.begin(), types.end());
     return success();
   }
+
+private:
+  /// Parse either an operand list or a region argument list depending on
+  /// whether `operand' is true.
+  ParseResult
+  parseOperandOrRegionArgList(SmallVectorImpl<OperandType> &result,
+                              bool isOperandList, int requiredOperandCount = -1,
```

Comment:
These default arguments should not be needed. This function is only invoked internally from others that already have the parameters specified.

Reply:
Thanks, done.